web_field_tooltip/readme/ROADMAP.rst

```diff
@@ -0,0 +1 @@
+* Find a way to update form views after tooltips update without refreshing manually
```

Comment:
Still true?

Reply:
yes but I dont think we should do something about it.